student/src/main/java/tds/student/services/ItemScoringService.java

```diff
@@ -69,7 +70,7 @@
 import tds.student.sql.singletons.ClientSingleton;
 import tds.student.tdslogger.TDSLogger;
 
-@Component
+@Service("legacyItemScoringService")
```

Comment:
We don't have to annotate classes with both \@Service and \@Component, Spring will register either one for DI.